versions.yaml

```diff
@@ -14,24 +14,38 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-
-_variables:
-  from9: &from9 gcr.io/google-appengine/debian9:latest
 versions:
+  - dir: '2/debian9/2.4'
+    repo: haproxy2
+    tags:
+      - '2.4.2-debian9'
```

Comment:
If this is based on debian9 please update `dir` and `tags` accordingly

Reply:
*debian10

Reply:
done